lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -335,26 +335,28 @@ public static function remove_insecure_properties( $theme_json ) {
 	);
 
 	/**
-	 * This converts selectors like '.wp-element-button, .wp-block-button__link'
-	 * to an array, so that the block selector is added to both parts of the selector.
+	 * Function that appends a sub-selector to a existing one.
 	 *
-	 * @param string $element The string with all the element's selectors.
-	 * @param string $selector The string we want to append to the selectors.
-	 * @param string $position The position we wand to append the selector in.
-	 * @return string element selector.
+	 * Given the compounded $selector "h1, h2, h3"
+	 * and the $to_append selector ".some-class" the result will be
+	 * "h1.some-class, h2.some-class, h3.some-class".
+	 *
+	 * @since 5.8.0
+	 * @since 6.1.0 Added append position.
+	 *
+	 * @param string $selector  Original selector.
+	 * @param string $to_append Selector to append.
+	 * @param string $position  A position sub-selector should be appended. Default: 'right'.
+	 * @return string
 	 */
-	private static function appendToSelector( $element, $selector, $position = 0 ) {
-		$element_selector = array();
-		$el_selectors     = explode( ',', $element );
-		foreach ( $el_selectors as $el_selector_item ) {
-			if ( 0 === $position ) {
-				$element_selector[] = $selector . $el_selector_item;
-			} else {
-				$element_selector[] = $el_selector_item . $selector;
-			}
+	protected static function append_to_selector( $selector, $to_append, $position = 'right' ) {
```

Comment:
I decided to use `left|right` because of the [`str_pad`](https://www.php.net/manual/en/function.str-pad.php), but we can also use `start|end`.